android/src/main/kotlin/com/shatsy/admobflutter/AdmobBanner.kt

```diff
@@ -14,18 +17,27 @@ import io.flutter.plugin.platform.PlatformView
 
 class AdmobBanner(context: Context, messenger: BinaryMessenger, id: Int, args: HashMap<*, *>?) : PlatformView, MethodCallHandler {
   private val channel: MethodChannel = MethodChannel(messenger, "admob_flutter/banner_$id")
-  private val adView: AdView = AdView(context)
+  private val adView: PublisherAdView = PublisherAdView(context)
```

Comment:
How does this change affect existing AdMob users? Does iOS have the equivalent class?

Reply:
on iOS there is a DFPBannerView.

I Don't have adMob account to test, so if you or someone could try this implementation and check if we can still load ads from admob would be great.

Both AdView and PublisherAdView extends ViewGroup
https://developers.google.com/android/reference/com/google/android/gms/ads/doubleclick/PublisherAdView
https://developers.google.com/android/reference/com/google/android/gms/ads/AdView


with PublisherAdView we have a little more power with our requests. For example we  can set more than one size to request to our adserver while with the simple AdView we can't